docs/content/manual/manual.yml

```diff
@@ -1726,6 +1726,17 @@ sections:
             input: '"useful but not for é"'
             output: '"USEFUL BUT NOT FOR é"'
 
+      - title: "`downcase`, `upcase`"
+        body: |
+
+          Emit a copy of the input string with its characters (unicode) converted to the
+          specified case.
+
+        example:
+          - program: 'upcase'
+            input: '"useful for é"'
+            output: '"USEFUL FOR É"'
```

Comment:
`output` of manual example needs to be an array (make sure to `make tests/man.test`)

Reply:
I was wondering why they did not show up in jq.1. These examples were just copied from the existing examples for `ascii_downcase` and `ascii_upcase`, so I suppose this comment would apply to those as well?

Reply:
Yes, those examples should be fixed as well.

Reply:
Those `ascii_upcase`/`ascii_downcase` examples as well as a bunch of other examples are not even generated because tehy are using `example:` instead of `examples:`.

You should put examples in the `examples:`, not `example:`

Reply:
is there any definition of the schema that manual.yml should conform to? If so (or if not, and it's easy to create), I will run a validator on it to check for any other issues (and the validation steps would be available for future reuse as well...)

Reply:
I added a schema for manual files in #2745, and validate both on building the manual locally and on GitHub Actions.